src/main/java/uk/gov/companieshouse/limitedpartnershipsapi/service/LimitedPartnershipService.java

```diff
@@ -98,7 +99,7 @@ public void updateLimitedPartnership(Transaction transaction,
         var lpSubmissionDaoAfterPatch = mapper.dtoToDao(lpSubmissionDto);
 
         // Need to ensure we don't lose the meta-data already set on the Mongo document (but lost when DAO is mapped to a DTO)
-        copyMetaDataForUpdate(submissionId, lpSubmissionDaoBeforePatch, lpSubmissionDaoAfterPatch);
+        copyMetaDataForUpdate(lpSubmissionDaoBeforePatch, lpSubmissionDaoAfterPatch);
```

Comment:
There was no need to pass `submissionId` in here - it's already available on the `lpSubmissionDaoBeforePatch` object.